src/views/skuLookup.tsx

```diff
@@ -0,0 +1,349 @@
+import { FC, KeyboardEvent, useEffect, useState } from "react";
+import useScreenSize from "../hooks/useScreenSize";
+import CustomCard from "../components/common/customcard";
+import SearchBar from "../components/common/searchbar";
+import Loader from "../components/loader";
+import {
+  SKU_LOOKUP_CATALOG,
+  SKU_LOOKUP_INVENTORY_STATUS,
+} from "../constants/apiConstants";
+import {
+  SCREEN_WIDTH,
+  SKU_DETAILS,
+  SKU_INVENTORY,
+  SKU_PRICE_DETAILS,
+  SKU_PRODUCT_INFO,
+} from "../constants/appConstants";
+import {
+  HorizontalTableProps,
+  HeaderProps,
+} from "../@types/components/commonTypes";
+import { SkuCardProps } from "../@types/pages/skuLookup";
+import { fetchData } from "../utils/fetchUtil";
+
+const HorizontalTable: FC<HorizontalTableProps> = (props) => {
+  const tableHeadColSpan =
+    props.tableData?.length > 0 &&
+    Object.entries(props.tableData[0]).length * 2;
+  return (
+    <div className="rounded-lg border border-black-400 overflow-x-auto mt-4">
+      <table className={`${props.tableClassName || ""} w-full`}>
```

Comment:
why is the Table component not being used ?

Reply:
Customtable component uses Primereact datatable component, which doesn't support horizontal structure. Hence html table is used